### PR TITLE
NP-45837 Map unisContent to Record

### PR DESCRIPTION
**Pull request by joachimjorgensen**

### CODE REVIEW

File: brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java [hunk 1]
```diff
@@ -30,6 +31,16 @@ public static boolean isValid(String value) {
         return false;
     }
 
+    public PublisherAuthority toPublisherAuthority() {
+        if (ACCEPTED.getValue().equals(this.getValue())) {
```
Comment (torbjokv) [inline brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java:35]: this == ACCEPTED

File: brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java [hunk 2]
```diff
@@ -30,6 +31,16 @@ public static boolean isValid(String value) {
...
+            return new PublisherAuthority(Collections.singleton("acceptedVersion"), false);
+        }
+        if (PUBLISHED.getValue().equals(this.getValue())) {
```
Comment (torbjokv) [inline brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java:38]: this == PUBLISHED

File: src/main/java/no/sikt/nva/UnisContent.java [hunk 3]
```diff
@@ -91,8 +109,25 @@ private void setFilename(String filename) {
...
+        record.setCristinId(Integer.toString(getCristinId()));
+        record.setResourceOwner(new ResourceOwner(UNIS_ID, new URI("")));
+        record.setPublisherAuthority(getPublisherAuthority().toPublisherAuthority());
+        record.setEntityDescription(new EntityDescription());
+        record.setType(new Type(Set.of(CRISTIN_RECORD.getValue()), CRISTIN_RECORD.getValue()));
+
+        var contentFile = new ContentFile(
+            getFilename(),
+            BundleType.ORIGINAL,
+            "",
```
Comment (torbjokv) [inline src/main/java/no/sikt/nva/UnisContent.java:124]: put in a variable with a name maybe?

File: src/main/java/no/sikt/nva/UnisContent.java [hunk 4]
```diff
@@ -91,8 +109,25 @@ private void setFilename(String filename) {
...
+            UUID.randomUUID(),
```
Comment (torbjokv) [inline src/main/java/no/sikt/nva/UnisContent.java:125]: great with a name here also (even if intellij does it for us)

**Review by torbjokv — APPROVED** [review]

File: brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java (hunk 1 above)
Comment (joachimjorgensen) [reply brage-migration-common/src/main/java/no/sikt/nva/brage/migration/common/model/record/PublisherAuthorityEnum.java:35]: På tide med ferie nå ja 🙈

**Pull request merged**